sdcm/cluster_gce.py

```diff
@@ -300,6 +301,13 @@ def _create_instance(self, node_index, dc_idx, spot=False):
         # Name must start with a lowercase letter followed by up to 63
         # lowercase letters, numbers, or hyphens, and cannot end with a hyphen
         assert len(name) <= 63, "Max length of instance name is 63"
+        startup_script = cluster.Setup.get_startup_script()
+        if "ubuntu-bionic" in self.params.get("scylla_linux_distro", ""):
```

Comment:
Does it cover both Debian 9 and Ubuntu 16,18?

Reply:
does the issue happen on ubuntu 16 and Debian 9?

Reply:
I checket: it happens only on Ubuntu 16, 18

Reply:
Are you sure??
I’m almost sure it happens on deb9

Reply:
Are you sure??
I’m almost sure it happens on deb9

Reply:
`sshguard` is not installed on Debian 9
```
Apr 27 13:09:59 rolling-upgrade-3-3-debian-stretch-db-node-d99b76c4-0-2 sshd[2575]: pam_unix(sshd:session): session closed for user scylla-test
Apr 27 13:10:04 rolling-upgrade-3-3-debian-stretch-db-node-d99b76c4-0-2 sshd[2585]: Accepted publickey for scylla-test from 10.142.0.5 port 52424 ssh2: RSA SHA256:ORclW5uOPUGIzX/HqWCBo6YH8GIz8D8MPjTEd0njNms
Apr 27 13:10:04 rolling-upgrade-3-3-debian-stretch-db-node-d99b76c4-0-2 sshd[2585]: pam_unix(sshd:session): session opened for user scylla-test by (uid=0)
Apr 27 13:10:04 rolling-upgrade-3-3-debian-stretch-db-node-d99b76c4-0-2 sshd[2585]: pam_unix(sshd:session): session closed for user scylla-test
/sshguard
Pattern not found
root@rolling-upgrade-3-3-debian-stretch-db-node-d99b76c4-0-2:/home/scylla-test# cat /etc/os
osconfig/   os-release  
root@rolling-upgrade-3-3-debian-stretch-db-node-d99b76c4-0-2:/home/scylla-test# cat /etc/os-release 
PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
NAME="Debian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
VERSION_CODENAME=stretch
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
root@rolling-upgrade-3-3-debian-stretch-db-node-d99b76c4-0-2:/home/scylla-test# systemctl status sshguard
Unit sshguard.service could not be found.
root@rolling-upgrade-3-3-debian-stretch-db-node-d99b76c4-0-2:/home/scylla-test# 

```